x-pack/metricbeat/module/azure/client_utils.go

```diff
@@ -198,3 +176,12 @@ func ContainsDimension(dimension string, dimensions []insights.LocalizableString
 	}
 	return false
 }
+
+func containsResource(resId string, resources []Resource) bool {
```

Comment:
It this actually client's utils or is could be re-organised to a generic `utils.go` for the whole `azure` module?

Reply:
it is clients_utils , would like to keep it that way for the moment as it might get too muddy if we add more functionality, once a function is reused at other levels as service etc then it needs rethinking.

Reply:
👍